ca_description/urdf/create_2.xacro

```diff
@@ -17,5 +17,5 @@
   </xacro:create_base>
 
   <!-- Roomblock -->
-  <xacro:roomblock parent="base_link" visualize="$(arg visualize)"/>
+  <!-- <xacro:roomblock parent="base_link" visualize="$(arg visualize)"/> -->
```

Comment:
Check if `roomblock` affects how the robot moves the block. If so, check how we can mitigate this effect.